### PR TITLE
added new device endpoint

### PR DESCRIPTION
**Pull request by tanneberger**
This endpoint returns a randomly chosen device.

### CODE REVIEW

File: app/models/device.py [hunk 1]
```diff
@@ -87,3 +88,15 @@ def check_access(self, user: str) -> bool:
             return True
 
         return m.contact_microservice("service", ["check_part_owner"], {"user_uuid": user})["ok"]
+
+    @staticmethod
+    def random(user: str) -> 'Device':
+
+        # Warning this only works for mysql
```
Comment (MaxiHuHe04) [inline app/models/device.py:94]: `func.random()` instead of `func.rand()` should supposedly also work for SQLite

Comment (tanneberger) [reply app/models/device.py:94]: Let me leave that here https://stackoverflow.com/questions/60805/getting-random-row-through-sqlalchemy -> func.rand() for mysql but youre right I should also add the option for sqllite for development purposes.

Comment (MaxiHuHe04) [reply app/models/device.py:94]: Have you tried it? https://stackoverflow.com/questions/60805/getting-random-row-through-sqlalchemy#comment68799157_60815

Comment (tanneberger) [reply app/models/device.py:94]: func.random() supports mysql and sqllite ? ...

**Comment by felixevers** [comment]
Already implemented.

**Pull request closed without merging**